use-shopping-cart/src/index.js

```diff
@@ -30,7 +30,7 @@ export const CartContext = createContext([
   () => {}
 ])
 
-export const CartProvider = ({
+export function CartProvider({
```

Comment:
Oh, I also did this because there was some issue that Sucrase was detecting where the built code would use the `this` keyword inside this function but arrow functions don't have access to a `this` keyword. To fix this I simply switched it to a regular `function` to grant access to the `this` keyword.